### ISSUE TITLE
Reintegrate lorcon (802.11 packet crafting)

### ISSUE THREAD

**Issue by Mister-X-** [rapid7/metasploit-framework#9186]
Lorcon and lorcon2 support was removed in 17ddbccc3474fcef85cf731b44c381315b47c0e3 because lorcon was not maintained and not functioning anymore. See also #3200 and #4850.

There is a fork that might be usable now (for recent driver though, but most injection-capable drivers are mac80211) and development seems active: https://github.com/kismetwireless/lorcon

Even though there are other ways to craft 802.11 packets, it would be nice to have support in Metasploit again.

**Comment by MAVProxyUser** [comment]
I work with the author... let me know how I can help, I actually was looking at bringing support in myself for the recent DJI Drone ID packets. They use Beacon Packets with vendor Information Element tags containing the drone serial number, roll, pitch, yaw. It should be trivial to spoof with Lorcon. 

http://blog.kismetwireless.net/2017/11/dji-uav-drone-id.html
https://github.com/kismetwireless/kismet/blob/master/kaitai_definitions/dot11_ie_221_dji_droneid.ksy
![dpkvlbquiaeo0ma](https://user-images.githubusercontent.com/2904396/33247363-994597e6-d2eb-11e7-8aa5-b9c46879172b.jpg)

**Comment by DJISDKUser** [comment]
This is my best attempt at addressing your question @Mister-X-  https://github.com/rapid7/metasploit-framework/pull/9301

**Comment by Fadila82** [comment]
I have started a branch from @DJISDKUser branch where I have started a ieee802.11 frame parser (and later crafter hopefully). 
For now, I have an example of probe request listener that only display the mac address of the sender and the number of probe requests received by it.
I would like to go further, but I'm new to github and I don't fully understand how to proceed.
Where to submit the merge request ? to @DJISDKUser branch ? Or should I wait for @DJISDKUser's branch to be merged, and then make my merge request ?
